HLTrigger/JetMET/src/HLTHPDFilter.cc

```diff
@@ -104,7 +106,7 @@ bool HLTHPDFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSetup) {
   iEvent.getByToken(m_theRecHitCollectionToken, hbhe);
 
   // collect energies
-  float hpdEnergy[4][73];
+  std::array<std::array<float, 73>, 4> hpdEnergy;
```

Comment:
IMHO this change obfuscates the code rather then making it more clear.  Wondering if there's not a better way to silence the compiler. Not clear btw why the warning is emitted.

Reply:
@mmusich there is a known "bug" with C-style arrays and LTO, where compiler emits false-positive warnings about accessing elements outside of array boundaries.